pkg/utils/namespace.go

```diff
@@ -0,0 +1,55 @@
+package utils
+
+import (
+	"os"
+
+	v1 "github.com/fission/fission/pkg/apis/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
```

Comment:
Let's add unit tests for this new code since it's critical for new functioning.

Reply:
done